crates/ethcore/src/spec/spec.rs

```diff
@@ -417,6 +444,15 @@ impl From<ethjson::spec::Params> for CommonParams {
             kip6_transition: p
                 .kip6_transition
                 .map_or_else(BlockNumber::max_value, Into::into),
+            eip1559_base_fee_max_change_denominator: p
```

Comment:
No real complaint, as it is used in only one place and checked for zero there, but an `Option<U256>` would really be preferable here. A divisor with default value zero is awkward.

Reply:
Yes, good point.
Fixed. Please check the fix.

Reply:
Checked it, lgtm